enhancements/insights/rapid-recommendations.md

```diff
@@ -0,0 +1,549 @@
+* [OpenshiftAuthenticationLogs](https://github.com/openshift/insights-operator/blob/master/docs/gathered-data.md#openshiftauthenticationlogs)
+* [OpenshiftSDNControllerLogs](https://github.com/openshift/insights-operator/blob/master/docs/gathered-data.md#openshiftsdncontrollerlogs)
+* [OpenshiftSDNLogs](https://github.com/openshift/insights-operator/blob/master/docs/gathered-data.md#openshiftsdnlogs)
+* [SAPVsystemIptablesLogs](https://github.com/openshift/insights-operator/blob/master/docs/gathered-data.md#sapvsystemiptableslogs)
+* [SchedulerLogs](https://github.com/openshift/insights-operator/blob/master/docs/gathered-data.md#schedulerlogs)
+
+
+### Risks and Mitigations
+
+#### Remote Configuration Service Inaccessible
+
+Risks:
+
+* The configured remote configuration service endpoint is invalid.
+* The remote configuration is not available temporarily (service down, network configuration issues).
+
+Mitigations:
+
+* The Insights Operator will keep a [local copy](#remote-configuration-processing) of the last used 
+  remote configuration in a config map and use it if it fails to download a new copy.
+* Red Hat will monitor the remote configuration service directly as well as the 
+  `RemoteConfigurationUnavailable` condition 
+  (see [Status Reporting and Monitoring](#status-reporting-and-monitoring)) 
+  in incoming Insights operator archives.
+
+In disconnected and air-gapped clusters, two behaviors can occur:
+* If data gathering or the container logs gatherer will be disabled, the Insights Operator will not 
+  attempt to download the configuration at all.
+* If data gathering is enabled or triggered manually, the Insights Operator will 
+  [report](#status-reporting-and-monitoring) the failure 
+  and [proceed](#remote-configuration-processing) with empty remote configuration.
+
+#### Invalid Remote Configuration
+
+Risk: The remote configuration cannot be parsed or does not pass schema validation.
+
+Mitigation: Red Hat will monitor the `RemoteConfigurationInvalid` condition 
+(see [Status Reporting and Monitoring](#status-reporting-and-monitoring)) 
+and fix/work around the issue by updating the remote configuration or the remote configuration service.
+
+#### Data Gathering Failures
+
+Risk: No data found/gathered for the specific configuration/request.
+
+Mitigation: This should not be a major issue and should only affect the relevant 
+Insights recommendation that requires the data. 
+he Insights archive metadata and also the `insightsoperator.operator.openshift.io` CR should indicate
+that no matching data was found in the cluster for a particular GVK request.
+
+#### Excessive Use of Cluster Resources
+
+Risk: Remote configuration causes the Insights Operator to process large amounts of data.
+
+Mitigation: The resource consumption will be [monitored](#status-reporting-and-monitoring). 
```

Comment:
Time should also be considered (I know it is in the metrics, but not sure if considered a resource)

Reply:
Computational time is considered a resource. The information about the time to gather is already available per gatherer in `gathers.json`.